apps/explorer/src/explorer/ExplorerApp.tsx

```diff
@@ -161,6 +162,7 @@ export const ExplorerApp: React.FC = () => {
   return (
     <CowAnalyticsProvider cowAnalytics={cowAnalytics}>
       <GlobalStyle />
+      <ScrollToHashElement />
```

Comment:
Changed!

Replaced `react-router-hash-link` with `@cascadia-code/scroll-to-hash-element`.
This is the setup for it.

Reply:
Do we need it? If I remember we used it only for FAQ which is not in the project anymore?

Reply:
Removed it and part of the markdown stuff as well.